src/Npgsql/NpgsqlDataReader.cs

```diff
@@ -2121,17 +2121,17 @@ FieldDescription GetInfo(int ordinal, Type? type, out PgConverter converter, out
 
         if (type is null)
         {
-            converter = field.ObjectOrDefaultInfo.Converter;
-            bufferRequirement = field.ObjectOrDefaultInfo.BufferRequirement;
-            asObject = field.ObjectOrDefaultInfo.AsObject;
+            var odfInfo = field.ObjectOrDefaultInfo;
```

Comment:
Records would add bloat :) I maybe could add a deconstruct.

Reply:
> Records would add bloat

Even if it's not actually used?

Reply:
Yeah because it adds equality code and impls of interfaces that won't be trimmed once any code uses that interface once (just like the GetValues() IEnumerable issue we have on DbDataReader).

Reply:
Thanks for the context, understand now.